Spanner/src/SnapshotTrait.php

```diff
@@ -68,6 +72,7 @@ private function initialize(
             : self::TYPE_SINGLE_USE;
 
         $this->context = SessionPoolInterface::CONTEXT_READ;
+        $this->directedReadOptions = $options['directedReadOptions'] ?? [];
```

Comment:
Can you help me understand what's the difference between and how would a user choose among these options?
1. I don't provide `directedReadOptions` in the `$options`
2. I provide `$options['directedReadOptions'] = null`
3. I provide `$options['directedReadOptions'] = []`
4. Some other way of passing empty config like: 
    `$options['directedReadOptions'] = ['include_replicas' => [], 'exclude_replicas' => []]`